graphene/relay/node.py

```diff
@@ -48,7 +48,15 @@ def get_resolver(self, parent_resolver):
 
 
 class NodeField(Field):
-    def __init__(self, node, type=False, deprecation_reason=None, name=None, **kwargs):
+    def __init__(
+        self,
+        node,
+        type=False,
+        deprecation_reason=None,
+        name=None,
+        description=None,
+        **kwargs
```

Comment:
Should this be dropped since nothing is being done with these kwargs anyway?

Reply:
Hmm.. this has to do with my original question. Is there any reason why we should not just pass `**kwargs` to Field's constructor?
For example, something like this:
```python
def __init__(self, node, type=False, **kwargs):
        assert issubclass(node, Node), "NodeField can only operate in Nodes"
        self.node_type = node
        self.field_type = type

        super(NodeField, self).__init__(
            type or node,
            id=ID(required=True, description="The ID of the object"),
            **kwargs
        )
```

Reply:
I don’t see why not. If nothing breaks then I think passing all the kwargs through makes sense. 

(Sorry for the delay in replying btw)

Reply:
Nothing breaks indeed. Let me know if anything else needs to be added.